zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -2741,6 +2881,49 @@ paths:
 
 components:
   schemas:
+    TenantCreateRequest:
+      type: object
+      properties:
+        tenantId:
+          type: string
+          description: The unique identifier of the tenant.
```

Comment:
We're using the same description here for `tenantId` and `tenantKey`. I don't have enough context to understand how they are related, but this property should likely have a different description.

Reply:
It's still confusing for me, but maybe that's because I don't know what the id is actually used for (and why we need it in addition to the key).